File: Optimize/New/New/ViewController.m
//
//  ViewController.m
//  New
//
//  Created by cyd on 2021/4/14.
//

#import "ViewController.h"
#import <objc/runtime.h>

@interface Persion : NSObject

@end

@implementation Persion



@end


@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    Persion *per = [Persion new];
    NSLog(@"%@",[per class]);
    NSLog(@"%@",object_getClass(per));

    NSLog(@"%@",[Persion class]);
//    NSLog(@"%@",object_getClass(Persion));

    
}


@end

File: MixedTest/FlutterIos/FlutterIos/ViewController.swift
//
//  ViewController.swift
//  FlutterIos
//
//  Created by cyd on 2021/3/8.
//

import UIKit
import Flutter
import FlutterPluginRegistrant
class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        self.navigationItem.title = "混合开发"
    }

    @IBAction func jumpNavite(_ sender: UIButton) {
        let native = NaviteController()
        self.navigationController?.pushViewController(native, animated: true)
    }
    
    @IBAction func jumpFlutter(_ sender: UIButton) {
        let flutter = FlutterViewController()
        self.navigationController?.pushViewController(flutter, animated: true)
    }
    
}

